app/components/pages/SubmissionWizard/steps/Submission/SubjectAreaDropdown.js

```diff
@@ -142,7 +153,7 @@ class SubjectAreaDropdown extends React.Component {
           ),
         },
         backgroundColor: this.props.theme.backgroundColor,
-        minHeight: this.props.gridUnit,
+        minHeight: `${gridUnitValue * 8}px`,
```

Comment:
This could be a `space.5` instead. I think we give preference to the `space` unit over a multiple of the `gridUnit` elsewhere in the app.

Reply:
In this file those are used as gridUnitValue in all the instances of that. I followed the pattern in the file.
all the keys in customReactSelectStyles use gridUnitValue instead. It is a definition of styles instead of a styled component. I think being consistent in this object will make more sense. In the styled definitions I am using space.

Reply:
I can change all of them though ?

Reply:
Please see the latest changes on that. Thank you